package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-aggregate-translations",
+  "version": "0.0.0",
+  "description": "The terra-aggregate-translations pre-build tool assists with creating the translation, intl loader and translation loader files that are configured for the specified locales. This tool is offered as a CLI script and as a setup function.",
+  "main": "lib/index.js",
```

Comment:
Not sure this will be right

Reply:
We can change it once we get source code in the repo.

Reply:
Removed main key, we can add it as we add source code to the project.
https://github.com/cerner/terra-aggregate-translations/pull/1/commits/054823863b04f4d041e4dbe7fefd3dd3471d7293